src/main/java/com/auth0/client/mgmt/filter/PageFilter.java

```diff
@@ -0,0 +1,31 @@
+package com.auth0.client.mgmt.filter;
+
+/**
+ * Class used to filter the results received when calling the Grants endpoint. Related to the {@link com.auth0.client.mgmt.RolesEntity} entity.
```

Comment:
not "Grants". Maybe a generic description will fit most of the (future) use cases. Something like 

```
Class used to filter the results received when calling an endpoint that supports Pagination.
```

Reply:
This has been implemented as suggested.